phpstan-config.neon

```diff
@@ -1,6 +1,11 @@
 parameters:
-    level: 1
+    level: 5
 
     paths:
         - src/
         - tests/
+
+    ignoreErrors:
+        -
+           message: '#Method Seld\JsonLint\JsonParser::lint\(\) should return Seld\JsonLint\ParsingException|null but return statement is missing.#'
```

Comment:
I would add the `return null;` instead of ignoring the error.

Reply:
fixed